modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/request_controller.rb

```diff
@@ -9,6 +10,26 @@ module Veterans
       class PowerOfAttorney::RequestController < ClaimsApi::V2::Veterans::PowerOfAttorney::BaseController
         FORM_NUMBER = 'POA_REQUEST'
 
+        def index
+          poa_codes = form_attributes['poaCodes']
+
+          unless poa_codes.is_a?(Array) && poa_codes.size.positive?
+            raise ::Common::Exceptions::ParameterMissing.new('poaCodes',
+                                                             detail: 'poaCodes is required and cannot be empty')
+          end
+
+          service = ManageRepresentativeService.new(external_uid: 'power-of-attorney-request',
+                                                    external_key: 'power-of-attorney-request')
```

Comment:
Thoughts on what the `uid` and `key` should be? Noting that we do not have a `target_veteran` in this request.

Reply:
Looks like a similar pattern was used for other services that did not have an obvious ID key to use.

```
... new(external_uid: 'find_poas_service_uid', ...
```

So I think this works.  The only part of that I did like from the `find_poa_service.rb` file example I found was the `-uid` and `-key` placed on the ends to mark them as different, but I'm not sure that would ever matter tbh.  Also personally, and this is really a **_nit_**, I would prefer to use underscores on these.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/18820/commits/35cb30e357285a8c755caf3aa5b98a19c05b4d5d